### DISCUSSION TITLE
Vertical Scrollbar in react table

### DISCUSSION THREAD

**Discussion by phani50** [TanStack/table#2744] [Q&A]
Hi,
To add scroll bar i have added the style for the React table like below,

<ReactTable
                            data={this.state.data}
                            columns={columns}
                            pages={this.state.pages}
                            defaultPageSize={this.state.defaultPageSize}
                            className="-striped -highlight"
                            loading={this.state.loading}
                            style={{
                                height:"300px"
                            }}
/>
So the problem is solved, it added the scrollbar.
But after this i have faced some issue, all the column data moved to left little bit. it is because of the scrollbar.
For me scrollbar should add but it should not move data because of this.
You can check close in attached image. Can you help me here ?
![react-table-issue](https://user-images.githubusercontent.com/1286048/93930206-a23c1b00-fd3a-11ea-86cf-fbe5045aecd7.PNG)

**Comment by tannerlinsley** [comment]
V6 support is deprecated. Please use the discussions tab to discuss deprecated versions.
